yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/table-index-iops.psql

```diff
@@ -12,12 +12,12 @@ SELECT
         ELSE 'unknown'
     END AS object_type,
     CASE 
-        WHEN c.relkind IN ('r', 'm') THEN psut.seq_tup_read
-        WHEN c.relkind = 'i' THEN psi.idx_scan
+        WHEN c.relkind IN ('r', 'm') THEN COALESCE(psut.seq_scan + psut.idx_scan, 0)
+        WHEN c.relkind = 'i' THEN 0::bigint -- just a placeholder so that sizing algorithm doesn't fail
```

Comment:
Pls check with Shahrukh that this won't break any assumptions that sizing algorithm has (i.e. we are reporting reads only on the table, not the corresponding indexes.

Reply:
yes already checked